File: Python3/프로그래머스/1/42748. K번째수/K번째수.py
def solution(array, commands):
    answer = []
    for com in commands:
        i = com[0] - 1
        j = com[1]
        k = com[2] - 1
        
        temp = array[i:j]
        temp.sort()
        answer.append(temp[k])
    return answer

File: JavaScript/프로그래머스/1/42748. K번째수/K번째수.js
function solution(array, commands) {
    var answer = [];
    var temp = [];
    let i,j,k;
    for (com of commands) {
        i = com[0] - 1;
        j = com[1];
        k = com[2] - 1;
        
        temp = array.slice(i, j);
        temp.sort((a,b) => a - b);
        answer.push(temp[k])
    }
    return answer;
}